client/view/index/Index.html

```diff
@@ -1,25 +1,37 @@
 <div class="container">
 
+	{#if message}
+		<alert on:animationend="this.setAttribute('class', 'hidden')" ref:alert>
```

Comment:
We should avoid making up new element types - `alert` isn't a [standardized HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element), so in this case it would be better to toss an `alert` class on the element instead.